packages/modelence/src/data/store.ts

```diff
@@ -64,6 +66,7 @@ export class Store<
   private readonly schema: TSchema;
   private readonly methods?: TMethods;
   private readonly indexes: IndexDescription[];
+  private readonly searchIndexes: SearchIndexDescription[];
```

Comment:
@omegascorp why do we need search indexes? Isn't this for full text search, as opposed to vector indexes, which is what we need?

Also, regardless of this question, I wanted to share something that I wrote a while ago in case it's helpful, since it's very related:
https://www.notion.so/AI-18d898b951c580d48d9fd647619e9234#18d898b951c580d48d9fd647619e9234

Reply:
@artahian I think it looks simpler when they are separated. They have a different structure, and search indexes are only applicable to the Atlas.